rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/report/SW360ReportController.java

```diff
@@ -184,27 +157,28 @@ private boolean validateMimeType(String mimeType) {
         return mimeTypeList.contains(mimeType);
     }
 
-    @RequestMapping(value = REPORTS_URL + "/download", method = RequestMethod.GET)
-    public void downloadExcel(HttpServletRequest request,HttpServletResponse response) throws TException{
-    	final User sw360User = restControllerHelper.getSw360UserFromAuthentication();
-    	String module = request.getParameter("module");
+    @GetMapping(value = REPORTS_URL + "/download")
+    public void downloadExcel(HttpServletRequest request, HttpServletResponse response) throws TException {
+        final User sw360User = restControllerHelper.getSw360UserFromAuthentication();
+        String module = request.getParameter("module");
         String token = request.getParameter("token");
         String extendedByReleases = request.getParameter("extendedByReleases");
-        User user=restControllerHelper.getUserByEmail(sw360User.getEmail());
-        String fileConstant="projects-%s.xlsx";
+        User user = restControllerHelper.getUserByEmail(sw360User.getEmail());
+        String fileConstant = "projects-%s.xlsx";
```

Comment:
Can be replaced by
`String fileConstant = module + "-%s.xlsx"`;

After that line 175 `fileConstant = "components-%s.xlsx";` can be removed

Reply:
@keerthi-bl, sorry but I mean replace line 167 by `String fileConstant = module + "-%s.xlsx";` and remove line 175 `fileConstant = "components-%s.xlsx";`

Reply:
Sorry, yes I updated the same.